File: Basics/BInarytoDecimal.cpp
#include<iostream>

using namespace std;

int main(){
    int number,mul=1,answer=0,remainder;
    cout << "Enter the binary number to find its decimal equivalent : ";
    cin >> number;
    while(number != 0){
        remainder = number % 10;
        number /= 10;
        answer = remainder * mul + answer;
        mul = mul * 2;
    }

    cout << "The Decinal equivalent is : " << answer;
}

File: Basics/BinarytoDecimal.java
import java.util.Scanner;

public class BinarytoDecimal {
    static int conversion(int number){
        int rem,mul=1,answer=0;
        while(number != 0){
            rem = number % 10;
            number /=10;
            answer = rem * mul + answer;
            mul = mul * 2;
        }
        return answer;
    }
    public static void main(String[] args) {
        int num = 1101;
        System.out.println(conversion(num));
    }
}
